calcom/cal.com
# Fixing spaces and wraps for event type legend

## What does this PR do?

Spaces and wraps for event type legend in the user profile needed some love.

<img width="317" alt="Mouse_Highlight_Overlay" src="https://user-images.githubusercontent.com/467258/171464719-945c9f21-9c59-47b6-83ee-cd93a9dab53e.png">

## Type of change


## How should this be tested?

Go to a user's profile page to see event type with legend such as price, recurring, etc. In smaller resolutions, legends should wrap.
